src/components/LandmarkDrawer.tsx

```diff
@@ -79,22 +91,65 @@ function Editor() {
   </>;
 }
 
-function LandmarkAssessmentHistory() {
-  return <Table>
+function LandmarkAssessmentHistory({ landmark } : {
+  landmark: Landmark;
+}) {  
+  const [user] = useUserContext();
+  const { data: assessments } = trpcNext.map.listLandmarkAssessment.useQuery({
+    userId: user.id,
+    landmark: landmark.名称,
+  });
+
+  const [selectedAssessment, setSelectedAssessment] = useState<LandmarkAssessment>();
+
+  return <>
+  <Table whiteSpace="nowrap">
     <Thead>历史评估结果</Thead>
     <Tr>
       <Th>日期</Th>
       <Th>结果</Th>
       <Th>评估人</Th>
       <Th>详情</Th>
     </Tr>
-    <Tbody> 
-    <Tr>
-      <Td>fake date</Td>
-      <Td>fake score</Td>  
-      <Td>fake accessor</Td>        
-      <Td><Link href="#">fake url</Link></Td>      
-    </Tr>   
+    <Tbody>
+      {assessments?.map((assessment, index) => (
+        <Tr key={index} onClick={() => setSelectedAssessment(assessment)} cursor="pointer">
+          <Td>{assessment.createdAt && prettifyDate(assessment.createdAt)}</Td>
+          <Td>{assessment.score}</Td>
+          <Td>假评估人</Td>
+          <Td>
+            <Text isTruncated 
+              maxWidth={{ base: mobileTextLimit, [sidebarBreakpoint]: desktopTextLimit }}>
+              {assessment.markdown}
+            </Text>      
+          </Td>
+        </Tr>
+      ))}
     </Tbody>
-  </Table>;
+  </Table>
+  {selectedAssessment && 
+      <AssessmentModal 
+        onClose={() => setSelectedAssessment(undefined)} 
+        assessment={selectedAssessment} />}
+  </>;
+}
+
+function AssessmentModal ({ onClose, assessment }: { 
+  onClose: () => void; 
+  assessment: LandmarkAssessment
+}) {          
+  return <ModalWithBackdrop isCentered isOpen onClose={onClose}>
+  <ModalContent>
+    <ModalCloseButton /> 
+    <ModalHeader>历史评估结果</ModalHeader>
+    <ModalBody>
+      <VStack gap={componentSpacing} align="left">
+        <p><b>日期：</b>{assessment.createdAt && prettifyDate(assessment.createdAt)}</p>
+        <p><b>结果：</b>{assessment.score}</p>
+        <p><b>评估人：</b>假评估人</p>
+        <p><b>详情：</b>{assessment.markdown || "无"}</p>
+      </VStack>
+    </ModalBody> 
+  </ModalContent>
```

Comment:
Add the standard model button "Close"

Reply:
![Screenshot 2024-10-03 at 10 42 12](https://github.com/user-attachments/assets/9c0eed95-4236-46cd-9853-3ad27ffb827c)

Reply:
![Uploading Screenshot 2024-10-03 at 10.47.35.png…]()